airflow/jobs.py

```diff
@@ -2234,13 +2235,23 @@ def _process_backfill_task_instances(self,
                                 # Skip scheduled state, we are executing immediately
                                 ti.state = State.QUEUED
                                 session.merge(ti)
+
+                                # no need to clean temp file, airflow run cli removes
+                                # it, see cli.py:342 - only relevant to executors running
+                                # locally
+                                cfg_path = None
```

Comment:
is this line needed?

Reply:
I believe so, otherwise `cfg_path` used further down in call `executor.queue_task_instance(...)` would not be defined if the flow does not go through the following `if` statement.